setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a 
```

Comment:
It's good that you documented what -n does but another common approach is to specify the longer version of the argument: --non-interactive